### ISSUE TITLE
Armor tab and separated key items

### ISSUE THREAD

**Issue by max641** [turtle-insect/MonsterHunterStories2#9]
Great work thank you!!
Don't know if is the right section.
Can you add the armor tab and if there is a flag on key item can you separate them in another section of item list?
Thanks

**Comment by HaroPla** [comment]
yeah, That would be good too. Also don't add or changes ANY key items, I hear that it can get you can banned online.
